src/core_modules/capture-core/components/LockedSelector/LockedSelector.actions.js

```diff
@@ -32,8 +32,8 @@ export const lockedSelectorBatchActionTypes = {
     ORG_UNIT_ID_RESET_BATCH: 'LockedSelector.BatchOrgUnitIdReset',
 };
 
-export const setOrgUnitFromLockedSelector = (id: string, orgUnit: Object) => actionCreator(lockedSelectorActionTypes.ORG_UNIT_ID_SET)({ id, orgUnit });
-export const setProgramIdFromLockedSelector = (id: string) => actionCreator(lockedSelectorActionTypes.PROGRAM_ID_SET)(id);
+export const setOrgUnitFromLockedSelector = (id: string, orgUnit: Object) => actionCreator(lockedSelectorActionTypes.ORG_UNIT_ID_SET)({ orgUnitId: id, orgUnit });
```

Comment:
NIT: maybe rename `setOrgUnitFromLockedSelector` input param instead?

Reply:
you mean to change the `id: string` on line 35 to be `orgUnitId` already?

Reply:
yep, doesn't make sense to you? (very nitty I know)

Reply:
Yep makes sense surely. Was only making sure I understood well